pydepsi/network.py

```diff
@@ -0,0 +1,178 @@
+"""Module for creating networks from STM points."""
+
+import math
+
+import numpy as np
+from scipy.spatial import Delaunay
+
+
+def get_distance(s, t):
+    """Calculate the distance between two points.
+
```

Comment:
Yes. Presumably, we will later add functionality to convert coordinates to a Euclidean system. This would, among other things, correct for rectangular grid cells (when using image coordinates) or curved axis (when using lon-lat).

This conversion may be done before creating the network, but there may be cases where this is not possible of desirable. In that case, the conversion could be done on-the-fly in this `get_distance` function.

Reply:
I added a comment to this effect in `get_distance`. I figured that this information may be important to retain for later reference, but it should not be exposed to the user.

I also renamed `get_distance` to `_get_distance` to mark it as a private function.

Reply:
thanks! I think this is good!